test-utils/docs/README.md

```diff
@@ -0,0 +1,156 @@
+## Tests
+
+### Run Tests
+
+In order to run all tests use command:
+
+```bash
+npm run test
+```
+
+To execute specific test(s), utilize the `-t` option followed by a string that must be present in the test path. This allows you to filter and run only the desired tests based on the specified criteria.
+
+```bash
+npm run test -t Image # runs tests for both BackgrounImage and Image
+
+npm run test -t /Image # runs a test for Image only
+```
+
+### Utils
+
+To perform component testing within the package, we employ the `@testing-library/react library`, which leverages the underlying functionality of `jest`.
```

Comment:
add link to react-testing-library docs `[@testing-library/react library](url)`

Reply:
fixed